bin/server.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (C) 2020 Vladislav Yarovoy. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Server-side of cusdub-tz. """
+
+import os
+
+from aiohttp import web
+
+from cdtz import config
+
+LIST_TZ = []
+
+
+def create_list_tz():
```

Comment:
Move the function to `cdtz/__init__.py` and import in the following way: `from cdtz import list_time_zones`.